src/logic/SystemPermissions.swift

```diff
@@ -62,10 +65,12 @@ class SystemPermissions {
     }
 
     private static func detectScreenRecordingIsGranted() -> PermissionStatus {
-        if #available(macOS 10.15, *) {
+        // ScreenCaptureKit is available from macOS 12.0
+        if #available(macOS 12.0, *) {
```

Comment:
We should probably have this change to the Screen Sharing permission check, in a separate Merge Request. It is unrelated to the goal of this current MR: Adopting Liquid Glass.